src/tests/subtract.rs

```diff
@@ -0,0 +1,369 @@
+use super::super::{Histogram, Counter, SubtractionError};
+use std::cmp;
+use std::borrow::Borrow;
+
+use num::Saturating;
+
+const TEST_VALUE_LEVEL: u64 = 4;
+
+fn assert_min_max_count<T: Counter, B: Borrow<Histogram<T>>>(hist: B) {
```

Comment:
moved tests that use this helper into an internal test module since it uses `value_for` which is now non-public. Plus, 300 lines of tests all on one function probably should have had its own file from the get-go.